components/mdc/icons/index.js

```diff
@@ -1,6 +1,9 @@
 import './_index.css'
+import { browser } from '$app/env'
```

Comment:
isn't this going to create a dep on kit?

Reply:
I think it's already included. I might need to test it in non kit stuff though.